### DISCUSSION TITLE
How to get endpoint of Azure Container App's custom domain?

### DISCUSSION THREAD

**Discussion by RamType0** [microsoft/aspire#5884] [Q&A] [answered]
In Aspire, We can get endpoint of services by `service.GetEndpoint(endpointName)`.

but `service.GetEndpoint("https")` always return default generated URL of Azure Container Apps.

How can we get endpoint of custom domain manually assigned to Azure Container App?

**Comment by davidfowl** [comment, marked as answer]
This isn't supported today.
